packages/ui/src/Customization/CustomizationsExtensionListItem.tsx

```diff
@@ -28,6 +29,54 @@ export interface IExtensionListItemProps {
 export class CustomizationsExtensionListItem extends React.Component<
   IExtensionListItemProps
 > {
+  public getDeleteTooltip() {
+    return (
+      <Tooltip id="deleteTip">
+        {this.props.i18nDeleteTip
+          ? this.props.i18nDeleteTip
+          : this.props.i18nDelete}
+      </Tooltip>
+    );
+  }
+
+  public getDetailsTooltip() {
+    return (
+      <Tooltip id="detailsTip">
+        {this.props.i18nDetailsTip
+          ? this.props.i18nDetailsTip
+          : this.props.i18nDetails}
+      </Tooltip>
+    );
+  }
+
+  public getUpdateTooltip() {
+    return (
+      <Tooltip id="updateTip">
+        {this.props.i18nUpdateTip
+          ? this.props.i18nUpdateTip
+          : this.props.i18nUpdate}
+      </Tooltip>
+    );
+  }
+
+  public handleDelete = () => {
```

Comment:
So, I prefer not using fat arrows to define class methods, because you don't get class methods this way, meaning that the method will not be present in the class prototype.

Is this a big deal? It depends, [this article](https://medium.com/@charpeni/arrow-functions-in-class-properties-might-not-be-as-great-as-we-think-3b3551c440b1) explains it in details.

As a rule of thumb, since this approach can lead to performance issues with many instances of the same component, I always write "normal" class methods and do the bind in the constructor manually. But that's me.

Reply:
I'll make that change. Also making same change in `CustomizationsApiConnectorListItem`. Also in `CustomizationsApiConnectorListItem` will make `render` that last method in the file.